### DISCUSSION TITLE
Cannnot reset HERO10 Overlays

### DISCUSSION THREAD

**Discussion by Chimchimgli** [gopro/labs#582] [General]
Been using GP Labs for about 2 weeks now, and really enjoying all these 'secret' features I've unlocked... don't remember exactly how I stumbled across Labs - probably from YouTube, but am I glad I found it.

There is one issue that I've experienced however, and that is when I livestream via RTMP through MonaServer... prior to installing the Labs firmware, I'd be able to livestream AND save the footage (that option to save as a 1080p file) no matter if I stream in 720p or 1080p. Now, I am only able to stream, and not save anything to my microSD card. I've tried using all my other SD cards, but it just won't save, except for the few seconds it does record - usually 16 seconds, before my GPHERO10 says the server or connection stopped.

I was thinking it might have to do with some of the overlays that I've scanned, maybe 150 Bitrate, or motion activation, there was quite a bit that I added. 

So anyway, is there something I can do to reset the camera to its original settings, or remove all the overlays, as I'd like to start fresh, knowing exactly what I want added now and not just scanning dozens of QR codes. And I have tried to factory reset the GoPro, as well as reformat my SD card(s), so I'm not really sure what else to do.

I would really appreciate any assistance. Thanks.

**Comment by dnewman-gpsw** [comment]
!RESET  removes any Labs enabled enhancements

![image](https://github.com/gopro/labs/assets/10856084/7a983679-0f0f-44f5-829b-41359a1d7e96)
